File: api/src/main/java/eu/suro/redis/RedisConfig.java
package eu.suro.redis;

import de.exlll.configlib.Comment;
import de.exlll.configlib.Configuration;
import lombok.Data;
import lombok.Getter;

@Configuration
@Data
public class RedisConfig {

    @Comment("Адрес сервера редиса")
    private String host = "localhost";
    @Comment("Порт сервера редиса")
    private int port = 6379;
    @Comment("Пароль сервера редиса")
    private String password = "";
}

File: api/src/main/kotlin/eu/suro/redis/RedisConfig.kt
package eu.suro.redis

import de.exlll.configlib.Comment

class RedisConfig {
    @Comment("Адрес сервера редиса")
    var host = "localhost"

    @Comment("Порт сервера редиса")
    var port = 6379

    @Comment("Пароль сервера редиса")
    var password = ""
}